### ISSUE TITLE
CSS rendering issue

### ISSUE THREAD

**Issue by abhishekalcanzar** [balderdashy/sails#3290]
I am getting some issue related to CSS rendering in sails, I think you might remember Olympus Project I am getting issue with respect to CSS rendering it does not loads css properly and complete layout seems be weired. can you please help me out as I am trying to debug this but not sure why it was breaking. 
![screenshot from 2015-10-08 17 30 48](https://cloud.githubusercontent.com/assets/6203409/10365619/5f8a4cec-6de2-11e5-96f6-73cbdba380e2.png)

Wherever I run it I usually get this issue, I had tried it on my localhost, on server with domain and at IP address also but no-where it seems to be working this is not a issue that occurs on each hit, it was occurring after several hits as like 1 in a 20 or more.

**Comment by sgress454** [comment]
Possible dupe of #2616?  See https://github.com/balderdashy/sails/issues/2616#issuecomment-139690258

**Comment by sailsbot** [comment]
Thanks for posting, @abhishekalcanzar.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
